test/nuget/TestApp/TestApp.vcxproj

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(ProjectDir)..\..\..\nuget\Microsoft.Windows.CppWinRT.props" />
```

Comment:
This path seems rather brittle. No better way to know where this is store? Sometimes it's relative to the project and sometimes to the solution, no?

Reply:
For the test app it doesn't matter too much as we 100% control where the file is located. I can make it relative to $(SolutionDir) though if you prefer.

Reply:
Ah, sorry I thought that was in the nuget files. Never mind.